pkg/api-server/testdata/inspect/meshgateways/_rules/meshhttproute.golden.json

```diff
@@ -0,0 +1,162 @@
+{
+ "httpMatches": [
+  {
+   "hash": "P6k5V5Y/QDNzTnF1vr7gY33PAAoDt+H+DtspboMfNMs=",
+   "match": [
+    {
+     "path": {
+      "value": "/api/v2",
+      "type": "PathPrefix"
+     }
+    }
+   ]
+  },
+  {
+   "hash": "waFsoIISmZDTfFWYqxvY265/GASHYEWvHQTwMh/bpuU=",
+   "match": [
+    {
+     "path": {
+      "value": "/api",
+      "type": "PathPrefix"
+     }
+    }
+   ]
+  }
+ ],
+ "resource": {
+  "mesh": "default",
+  "name": "the-gateway",
+  "type": "MeshGateway"
+ },
+ "rules": [
+  {
+   "fromRules": [],
+   "toRules": [
+    {
+     "conf": {
+      "Rules": [
+       {
+        "matches": [
+         {
+          "path": {
+           "value": "/api/v2",
+           "type": "PathPrefix"
+          }
+         }
+        ],
+        "default": {
+         "backendRefs": [
+          {
+           "kind": "MeshServiceSubset",
+           "name": "backend_kuma-demo_svc_3001",
+           "tags": {
+            "version": "1.0"
+           }
+          }
+         ]
+        }
+       },
+       {
+        "matches": [
+         {
+          "path": {
+           "value": "/api",
+           "type": "PathPrefix"
+          }
+         }
+        ],
+        "default": {
+         "backendRefs": [
+          {
+           "kind": "MeshServiceSubset",
+           "name": "other-svc",
+           "tags": {
+            "version": "1.0"
+           }
+          },
+          {
+           "kind": "MeshServiceSubset",
+           "name": "other-svc-2",
+           "tags": {
+            "version": "1.0"
+           }
+          }
+         ]
+        }
+       }
+      ]
+     },
+     "matchers": [],
+     "origin": [
+      {
+       "mesh": "default",
+       "name": "the-http-route",
+       "type": "MeshHTTPRoute"
+      },
+      {
+       "mesh": "default",
+       "name": "the-other-http-route",
+       "type": "MeshHTTPRoute"
+      }
+     ]
+    }
+   ],
+   "type": "MeshHTTPRoute",
+   "warnings": []
+  },
+  {
+   "fromRules": [],
+   "toRules": [
+    {
+     "conf": {
+      "http": {
+       "requestTimeout": "15s"
+      }
+     },
+     "matchers": [
+      {
+       "key": "__rule-matches-hash__",
+       "not": false,
+       "value": "P6k5V5Y/QDNzTnF1vr7gY33PAAoDt+H+DtspboMfNMs="
+      },
+      {
+       "key": "kuma.io/service",
+       "not": false,
+       "value": ""
```

Comment:
@michaelbeaumont this doesn't really added to me. Feels like this empty service shouldn't exist right? Is this a problem in my resource code or a more general policy matching issue?

Reply:
the `on-route` policy:

```yaml
type: MeshTimeout
name: on-route
mesh: default
spec:
  targetRef:
    kind: MeshHTTPRoute
    name: the-http-route
  to:
    - targetRef:
        kind: Mesh
      default:
        http:
          requestTimeout: 15s
```

Interesting that it wants to add a service match in there...